logbook-core/src/main/java/org/zalando/logbook/BodyFilters.java

```diff
@@ -69,7 +69,7 @@ public static BodyFilter replaceJsonStringProperty(final Set<String> properties,
         final Predicate<String> json = MediaTypeQuery.compile("application/json", "application/*+json");
 
         final String regex = properties.stream().map(Pattern::quote).collect(joining("|"));
-        final Pattern pattern = Pattern.compile("(\"(?:" + regex + ")\"\\s*:\\s*)\".*?\"");
+        final Pattern pattern = Pattern.compile("(\"(?:" + regex + ")\"\\s*:\\s*)\".*?(?<!\\\\)\"");
```

Comment:
```
"password": "abc\\","foo":"!?$123"
```
Hmmm :see_no_evil:

Reply:
We may want to offer an alternative `ParsingJsonBodyFilter` which allows filtering for field names (by regex? by path? by predicate?). The default `replaceJsonStringProperty` should then document its pitfalls.